project_task_dependencies/models/project_task.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2016 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    task_ids = fields.Many2many(
+        comodel_name='project.task', relation='project_task_project_task_rel',
+        column1='task_id', column2='dependent_task_id', string='Dependencies',
```

Comment:
This is not clear to me: the are the Tasks I depend on, or the Tasks the depend on me?
The field name (and string label) should make this clear.